### ISSUE TITLE
Changing focused surface does not update modifiers.

### ISSUE THREAD

**Issue by Ongy** [swaywm/wlroots#389]
When the focused view (in rootston) is changed while a modifier is held, e.g. close a shell+terminal with `Ctrl+d` and focus changes to previous view, the modifiers are not updated.

The easiest way to fix this is probably to send the current modifiers when the `wlr_seat` focus is changed. Either from the compositor or from inside wlroots itself.

**Comment by acrisci** [comment]
Yeah I think this is a wlr-seat concern to make sure the window that gets keyboard focus has the correct modifier state.
